packages/frontend/src/App.test.jsx

```diff
@@ -1,18 +1,45 @@
-import { setupTests } from 'helpers/tests';
+import { createSetupTestsForRoute } from 'helpers/tests';
 import App from 'App';
+import { baseState } from 'contexts';
+import { produce } from 'immer';
 
 describe('App', () => {
-  describe('routing', () => {
-    it('renders the Setup page at /setup', () => {
-      const { screen } = setupTests(App, { route: '/setup' });
-      expect(
-        screen.getByRole('banner', { name: /game setup/i })
-      ).toBeInTheDocument();
+  it('it renders that game component at /play', () => {
+    const setupTests = createSetupTestsForRoute('/play');
+
+    const { screen } = setupTests(App);
+
+    expect(screen.getByTestId('game-component')).toBeInTheDocument();
+  });
+
+  it('it renders the setup component at /setup', () => {
+    const setupTests = createSetupTestsForRoute('/setup');
+
+    const setupGameState = produce(baseState.game, (draft) => {
+      draft.stage = 'SETUP';

```

Comment:
All references to the game stage could be pulled from the `GAME_STAGES` object

Reply:
@perennialAutodidact change made.